### PR TITLE
Improve metrics description for journal metrics

### PR DESCRIPTION
**Pull request by yuzhu**
This improves and clarifies metrics description for journal metrics.

### CODE REVIEW

File: core/common/src/main/java/alluxio/metrics/MetricKey.java [hunk 1]
```diff
@@ -335,13 +335,17 @@ public MetricKey build() {
           .build();
   public static final MetricKey MASTER_JOURNAL_SPACE_FREE_BYTES =
       new Builder("Master.JournalFreeBytes")
-          .setDescription("Bytes left on the journal disk(s) for an Alluxio master")
+          .setDescription("Bytes left on the journal disk(s) for an Alluxio master. "
+              + "This metric is only valid on Linux and when embedded journal is used. "
```
Comment (ggezer) [inline core/common/src/main/java/alluxio/metrics/MetricKey.java:339]: nit: emphasize this with hypens?

File: core/common/src/main/java/alluxio/metrics/MetricKey.java [hunk 2]
```diff
@@ -335,13 +335,17 @@ public MetricKey build() {
...
+              + "Use this metric to monitor whether your journal is running out of disk space.")
           .setMetricType(MetricType.GAUGE)
           .build();
   public static final MetricKey MASTER_JOURNAL_SPACE_FREE_PERCENT =
       new Builder("Master.JournalFreePercent")
-          .setDescription(
-              "Percentage of free space left on the journal disk(s) for an Alluxio master")
+          .setDescription("Percentage of free space left on the journal disk(s) "
+              + "for an Alluxio master"
```
Comment (ggezer) [inline core/common/src/main/java/alluxio/metrics/MetricKey.java:346]: nit: missing dot at the end.

File: core/common/src/main/java/alluxio/metrics/MetricKey.java [hunk 3]
```diff
@@ -610,7 +614,11 @@ public MetricKey build() {
           .build();
   public static final MetricKey MASTER_UFS_JOURNAL_CATCHUP_TIMER =
       new Builder("Master.UfsJournalCatchupTimer")
-          .setDescription("The timer statistics of journal catchup")
+          .setDescription("The timer statistics of journal catchup"
+              + "Only valid when ufs journal and zookeeper is used. "
```
Comment (ggezer) [inline core/common/src/main/java/alluxio/metrics/MetricKey.java:618]: zookeeper seems redundant, as it's a prerequisite of using ufs journal anyway.

File: core/common/src/main/java/alluxio/metrics/MetricKey.java [hunk 4]
```diff
@@ -620,7 +628,11 @@ public MetricKey build() {
           .build();
   public static final MetricKey MASTER_UFS_JOURNAL_INITIAL_REPLAY_TIME_MS =
       new Builder("Master.UfsJournalInitialReplayTimeMs")
-          .setDescription("The process time of the ufs journal initial replay")
+          .setDescription("The process time of the ufs journal initial replay."
+              + "Only valid when ufs journal is used."
+              + " It records the time it took for the very first journal replay. "
+              + "It can be used to to determine if journal replay is the reason "
```
Comment (ggezer) [inline core/common/src/main/java/alluxio/metrics/MetricKey.java:634]: This sounds very specific.
Can you say something like above: "Use this metric to monitor when your master boot-up time is high."

File: core/common/src/main/java/alluxio/metrics/MetricKey.java [hunk 5]
```diff
@@ -343,7 +343,7 @@ public MetricKey build() {
   public static final MetricKey MASTER_JOURNAL_SPACE_FREE_PERCENT =
       new Builder("Master.JournalFreePercent")
           .setDescription("Percentage of free space left on the journal disk(s) "
-              + "for an Alluxio master"
+              + "for an Alluxio master。"
```
Comment (ggezer) [inline core/common/src/main/java/alluxio/metrics/MetricKey.java:346]: nice dot.

**Review by ggezer — APPROVED** [review]

**Comment by HelloHorizon** [comment]
alluxio-bot, merge this please

**Comment by HelloHorizon** [comment]
alluxio-bot, cherry-pick this to branch-2.6 please

**Pull request merged**